adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -446,7 +479,7 @@ private void handleConnectionRequestResult(final MqttEndpoint endpoint,
                         checkDeviceRegistration(authenticatedDevice, currentSpan.context()))
                         .map(ok -> authenticatedDevice))
                 .compose(authenticatedDevice -> createLinks(authenticatedDevice, currentSpan))
-                .compose(authenticatedDevice -> registerHandlers(endpoint, authenticatedDevice))
+                .compose(authenticatedDevice -> registerHandlers(endpoint, authenticatedDevice, traceSamplingPrio.result()))
```

Comment:
I would prefer `priority` over `prio`.

Reply:
I've renamed that accordingly in the class.